sqlmesh/core/node.py

```diff
@@ -71,7 +72,7 @@ def is_day(self) -> bool:
 
     @property
     def is_hour(self) -> bool:
-        return self == IntervalUnit.HOUR
+        return self in (IntervalUnit.HOUR, IntervalUnit.SIX_HOUR)
```

Comment:
I don't think this is right. I think we need a separate `is_six_hour` property method here, but since we don't have a use case for it we can add it later if needed. In any case `is_hour` shouldn't be updated